src/beast/beast/module/core/maths/Muldiv.cpp

```diff
@@ -0,0 +1,61 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of Beast: https://github.com/vinniefalco/Beast
+    Copyright 2013, Vinnie Falco <vinnie.falco@gmail.com>
+
+    Portions of this file are from JUCE.
```

Comment:
Portions arent' from JUCE and I think Muldiv belongs in Ripple, I would just tuck it into `STAmount.h` and we can worry about it later.